Musico/app/controllers/application_controller.rb

```diff
@@ -1,2 +1,12 @@
 class ApplicationController < ActionController::Base
+    include ApplicationHelper
```

Comment:
why we are including ApplicationHelper here?, helpers are used for different purpose. create a concern if you want to.

Reply:
And if you decide to create a concern also move this check_login method also there